odoo/addons/base/models/ir_actions.py

```diff
@@ -7,6 +7,7 @@
 from odoo.osv import expression
 from odoo.tools.safe_eval import safe_eval, test_python_expr
 from odoo.tools.float_utils import float_compare
+from odoo.http import request
```

Comment:
:+1: this import looks better than adding one in `odoo/models.py`

Reply:
https://github.com/odoo/odoo/blob/1c89dd90937c8b313397c4d91ee1f1006686d01e/odoo/models.py#L1459

:slightly_smiling_face:

Reply:
The purpose was to explicitly avoid import cycles at the file level.